NBA-Stats-Dashboard/src/index.css

```diff
@@ -3,7 +3,7 @@
   --beige: #f5f5dc;
   --white: #ffffff;
   --blue: #10448d;
-  --gray: #e5e5e5;
+  --gray: #f9f9f9;
```

Comment:
In my branch I have been using the previous gray #e5e5e5 and I consider this a true gray. The new one you overwrote is a very light gray. Do not delete they previous gray #e5e5e5, instead rename the gray you used in this pr #f9f9f9 to --light-gray